### PR TITLE
style: fix timeline custom dot props not align

### PR DESCRIPTION
**Pull request by chenc041**
[中文模板 / Chinese Template](https://github.com/DouyinFE/semi-design/blob/main/.github/PULL_REQUEST_TEMPLATE.zh-CN.md)

- [x] I have read and followed [Pull Request Guidelines](https://github.com/DouyinFE/semi-design/blob/main/CONTRIBUTING-en-US.md#pull-request-guidelines) of the contributing guide.

### What kind of change does this PR introduce? (check at least one)

 - [x] Bugfix
 - [ ] Feature
 - [ ] Code style update
 - [ ] Refactor
 - [ ] Test Case
 - [ ] TypeScript definition update
 - [ ] Document improve
 - [ ] CI/CD improve
 - [ ] Branch sync
 - [ ] Other, please describe:

### PR description

Fixes timeline custom dot property does not align with another dot

截屏2021-12-11 20 03 14

### Changelog
🇨🇳 Chinese
- 修复  timeline dot 不对齐

---

🇺🇸 English
- Fix  timeline custom dot property does not align with another dot

### Checklist
- [ ] Test or no need
- [ ] Document or no need
- [ ] Changelog or no need

### CODE REVIEW

**Comment by zwlafk** [comment]
这里不太符合预期了
https://codesandbox.io/s/semi-design-simple-story-forked-qr9mz?file=/src/App.jsx
image

File: packages/semi-foundation/timeline/timeline.scss [hunk 1]
```diff
@@ -111,6 +97,22 @@ $module: #{$prefix}-timeline;
         }
     }
 
+    &-alternate {
```
Comment (zwlafk) [inline packages/semi-foundation/timeline/timeline.scss:100]: 实际上有四种mode：`alternate | left | right | center`，这里把其他mode的样式取消掉了

**Comment by chenc041** [comment]
> 这里不太符合预期了 https://codesandbox.io/s/semi-design-simple-story-forked-qr9mz?file=/src/App.jsx image

OK, I review again

**Comment by zwlafk** [comment]
@chenc041 I pushed a new commit on this PR. You can start storybook by run `npm start` and have a check~

**Comment by chenc041** [comment]
> @chenc041 I pushed a new commit on this PR. You can start storybook by run `npm start` and have a check~

ok~

**Comment by chenc041** [comment]
> @chenc041 I pushed a new commit on this PR. You can start storybook by run `npm start` and have a check~

all mode is correct 🆒

**Pull request merged**